Automattic/jetpack
# Search: Translate copy for the new pricing table

Fixes #26594.

#### Changes proposed in this Pull Request:

Wraps the all strings for the new pricing table in a translation call.

#### Other information:


#### Jetpack product discussion


None.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Apply this change to a site without a Jetpack Search subscription.
* Open the new onboarding page via `/wp-admin/admin.php?page=jetpack-search&new_pricing_202208=1`.
* Ensure all the text render as expected.
